File: sudoku-game/game-boot/src/main/java/com/zumin/sudoku/game/mapper/GameLevelMapper.java
package com.zumin.sudoku.game.mapper;

import com.baomidou.mybatisplus.core.mapper.BaseMapper;
import com.zumin.sudoku.game.pojo.entity.GameLevel;
import org.apache.ibatis.annotations.Mapper;

@Mapper
public interface GameLevelMapper extends BaseMapper<GameLevel> {
}

File: sudoku-game/game-boot/src/main/java/com/zumin/sudoku/game/mapper/GameLevelMapper.kt
package com.zumin.sudoku.game.mapper

import com.baomidou.mybatisplus.core.mapper.BaseMapper
import com.zumin.sudoku.game.pojo.GameLevel
import org.apache.ibatis.annotations.Mapper

@Mapper
interface GameLevelMapper : BaseMapper<GameLevel>